Description: armel is no longer supported in Ubuntu, disable neon
 .
 qt4-x11 (4:4.8.4+dfsg-0ubuntu2) raring; urgency=low
 .
   * Disable neon for Ubuntu since armel is no longer supported.
Author: Tim Gardner <tim.gardner@canonical.com>

--- /dev/null
+++ qt4-x11-4.8.4+dfsg/imports/junk
@@ -0,0 +1 @@
+1
--- /dev/null
+++ qt4-x11-4.8.4+dfsg/templates/junk
@@ -0,0 +1 @@
+1
